theano/link/jax/__init__.py

```diff
@@ -0,0 +1 @@
+﻿from theano.link.jax.jax_linker import JAXLinker
```

Comment:
why not `.linker`?

Reply:
had this first, but then I noticed that the `compile` module is also the imperative 🤓